buildSrc/src/main/java/Dependencies.kt

```diff
@@ -148,8 +154,11 @@ object Dependencies {
     }
 
     object Protobuf {
+        @Deprecated("A new implementation is avaliable. Use the protobuf project instead.")
         const val wireJvmMessageProto = "com.wire:generic-message-proto:${Versions.wireJvmMessageProto}"
+        @Deprecated("A new implementation is avaliable. Use the protobuf project instead.")
```

Comment:
Also delete?

Reply:
Same as above. Still used for envelope creation.